### DISCUSSION TITLE
How to disable highlight

### DISCUSSION THREAD

**Discussion by hawk90** [NvChad/NvChad#1440] [Q&A] [answered]
<img width="801" alt="스크린샷 2022-08-07 오후 10 14 46" src="https://user-images.githubusercontent.com/6481112/183293711-52f3f740-0e49-4f0c-bf9a-af0207d2507c.png">

I wanna disable highlight match bracket.

**Comment by siduck** [comment, marked as answer]
just set these to false in blankline config i.e override blankline config 

![image](https://user-images.githubusercontent.com/59060246/183320704-62d7977c-16d1-42a1-a330-57e65ebc5413.png)

**Reply by hawk90** [reply]
Thx a lot. :)
